src/views/v2/month/calendar-body/day.php

```diff
@@ -55,6 +55,17 @@
 
 // Only add id if events exist on the day.
 $mobile_day_id = 'tribe-events-calendar-mobile-day-' . $day['year_number'] . '-' . $day['month_number'] . '-' . $day['day_number'];
+
+$events_label_singular = tribe_get_event_label_singular();
+$events_label_plural   = tribe_get_event_label_plural();
+
+$num_events_label = sprintf(
+	/* translators: %1$s: number of events, %2$s: Event (singular), %3$s: Events (plural). */
+	_n( '%1$s %2$s', '%1$s %3$s', $day['found_events'], 'the-events-calendar' ),
```

Comment:
:warning: Mismatched placeholders is probably an error

<sub>hash: c90961326cd3ad605e454d1729393a58</sub>